connectors/tests/test_byoc.py

```diff
@@ -337,7 +331,86 @@ async def test_heartbeat(mock_responses, patch_logger):
 
 
 @pytest.mark.asyncio
-async def test_connectors_get_list(mock_responses):
+async def test_connectors_get_list_no_connector(mock_responses):
+    config = {"host": "http://nowhere.com:9200", "user": "tarek", "password": "blah"}
+    connectors = ConnectorIndex(config)
+    conns = []
+    async for connector in connectors.supported_connectors():
+        conns.append(connector)
+
+    assert len(conns) == 0
+    await connectors.close()
```

Comment:
```suggestion
    await connectors.close()
    
    assert len(conns) == 0
```

I'd just flip them around cause asserts should be last in each individual test